cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt

```diff
@@ -132,14 +132,23 @@ datasource "azure_instances" do
     end
 end
 
+datasource "datasource_ds_azure_pricing" do
+  request do
+    host "raw.githubusercontent.com"
+    path "/flexera-public/policy_templates/master/data/azure/azure_vm_pricing.json"
+    header "User-Agent", "RS Policies"
+  end
+end
+
 datasource "filtered_resources" do
-  run_script $js_filter_resources, $azure_instances, $param_exclusion_tag_key
+  run_script $js_filter_resources, $azure_instances, $param_exclusion_tag_key, $datasource_ds_azure_pricing
 end
 
 ##########
```

Comment:
Datasources and Scripts should now be in a single combined section. I realize you didn't make it this way, but this is one of those things where, cuz you're touching the policy, it's yours now to fix :)

https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices#Basic-Policy-Structure